.github/workflows/release-pypi.yml

```diff
@@ -0,0 +1,79 @@
+---
+name: Release to PyPI
+
+on:
+  release:
+    types: [created]
+
```

Comment:
I don't know much of github action. I assume this means that the workflow runs whenever we push a tag?

Reply:
You can do that, but I think this method triggers when you create a release from the github UI.

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release

I think doing it on tag push would look like this

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags